scania-digital-design-system/tegel
# feat (accordion): add prop for removing the last accordion item bottom border

## **Hide last border in accordion**  
Add prop for removing the last accordion item bottom border.

## **Issue Linking:**  
https://github.com/scania-digital-design-system/tegel/issues/625

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Run Storybook.
2. Go the Accordion and select the Docs tab.
3. Select "Hide last border".
4. Go to the Canvas tab to see the result.

## **Checklist before submission**

## **Suggested test steps**
- [x] Added e2e-test

## **Screenshots**  
Before:
<img width="879" alt="before" src="https://github.com/scania-digital-design-system/tegel/assets/1917147/d366d3bb-3dbc-49c6-8e78-ef96054e961a">

After:
<img width="957" alt="after" src="https://github.com/scania-digital-design-system/tegel/assets/1917147/13d6e62d-880e-4df6-990a-acd8f82bd075">
